connexion/operations/swagger2.py

```diff
@@ -291,7 +290,7 @@ def _transform_form(self, form_parameters: t.List[dict]) -> dict:
 
             default = param.get("default")
             if default is not None:
-                defaults[param["name"]] = default
+                prop["default"] = default
```

Comment:
Also needs to be added for each property separately, so if a body is sent without the property, the default value is added.